docs/hpc/applications/easybuild.md

```diff
@@ -38,195 +198,249 @@ archspec/0.1.4-GCCcore-11.3.0                            SCOTCH/7.0.1-iimpi-2022
+  8) libxml2/2.11.5-GCCcore-13.2.0     19) FlexiBLAS/3.3.1-GCC-13.2.0       30) libffi/3.4.4-GCCcore-13.2.0     41) GROMACS/2024.1-foss-2023b
+  9) libpciaccess/0.17-GCCcore-13.2.0  20) FFTW/3.3.10-GCC-13.2.0           31) Python/3.11.5-GCCcore-13.2.0
+ 10) hwloc/2.9.2-GCCcore-13.2.0        21) gompi/2023b                      32) gfbf/2023b
+ 11) OpenSSL/1.1                       22) FFTW.MPI/3.3.10-gompi-2023b      33) cffi/1.15.1-GCCcore-13.2.0
 ```
 
-Note: As evident from the example above, apart from the "gateway" module "tools/dev" only the module for the software you want to use is required to be loaded. Any other module which is required will be loaded now automagically. This is different from the old system where you might need to load a compiler and MPI module. The new system that does not required this as the requirements for that software will be loaded automatically.
-Also, if you want to use two different programs at the same time, say A and B, you need to make sure you are using the same toolchain. In the example above, the toolchain would be foss and the version would be 2021b. Thus, you only could use B/1.2.3-foss-2021b but not B/1.2.3-foss-2022a together with GROMACS/2021.5-foss-2021b example from above!
+Note: As evident from the example above, apart from the "gateway" module`tools/dev` only the module for the software you want to use is required to be loaded. Any other module which is required will be loaded now automagically. It does not print out which modules are loaded, per default `ml` *silently* loads the required modules. Also, only the module of the software you want to use is required, all dependencies are automagically resolved. Furthermore, the `-foss-2023b` is what is called a *toolchain*. You only can load software within the same toolchain hierarchy, you cannot mix-and-match. See [here](https://docs.easybuild.io/common-toolchains/#toolchains_diagram) for more information.
 
 ### Within a Batch Job
 
 Using the development software stack within a batch job is the same as for using it on the login node. For example to load GROMACS 2021.5 from the development stack and use within a batch you, you could do the following:
 
 Using GROMACS 2021.5 from the development software stack in a batch job.
 
-```console
+```bash
 #PBS -lwalltime=01:00:00
 #PBS -lselect=1:ncpus=16:mem=16gb
 
-module purge
-module load tools/dev
-module load GROMACS/2021.5-foss-2021b
+ml purge
+ml tools/dev
+ml GROMACS/2021.5-foss-2021b
+# optional
+ml
 
 mpirun gmx_mpi <your options>
 ```
 
-## Using the Production Software Stack
+This would purge any loaded modules, loads the production software stack, loads the requested program, in this case GROMACS-2024.1 and optional lists all loaded modules. The last ml is only to document which other modules are loaded which might be of interest if other software is used at the same time!
```

Comment:
```suggestion
This would purge any loaded modules, loads the development software stack, loads the requested program, in this case GROMACS-2024.1 and optional lists all loaded modules. The last ml is only to document which other modules are loaded which might be of interest if other software is used at the same time!
```

Reply:
It should be `tools/prod` not `tools/dev`. I will have to changed that.